content/en/docs/signing_and_verification.md

```diff
@@ -0,0 +1,214 @@
+- **Domain Name**: Acquire and register a domain name for your Harbor registry. For example, you may choose `myharbor-registry.online` as the domain. 
+- **DNS Configuration**: Ensure that the domain points to the IP address of the Harbor server by configuring an appropriate DNS record. 
+
+#### 2. Install Certbot for SSL Certificates 
+Certbot, a tool from Let’s Encrypt, will be used to obtain a free SSL certificate for securing Harbor's domain. 
+
+- **Install Certbot**: 
+```bash
+sudo apt install --classic certbot
+
+**Obtain the SSL Certificate**: 
+
+Run the following command to obtain an SSL certificate using Certbot: 
+
+```bash
+sudo certbot certonly --standalone -d myharbor-registry.online
+
+# Building and Pushing Docker Images to Harbor
+
+After Harbor is set up and Notation is installed, you can begin building and pushing Docker images to your Harbor registry:
+
+## 1. Build the Image:
+- Create a Docker image from a repository and tag it with your Harbor registry domain:
+```bash
+docker build -t myharbor-registry.online/notation-project/net-monitor:v1 https://github.com/wabbit-networks/net-monitor.git#main
+```
+
+## 2. Push the Image to Harbor:
+- Log in to your Harbor registry:
+```bash
+docker login myharbor-registry.online
+```
+- Push the image to Harbor:
+```bash
+docker push myharbor-registry.online/notation-project/net-monitor:v1
+```
+- After pushing the image, you can check the "Signed" section in Harbor, which will indicate if the image is signed or unsigned.
+
+# Signing and Verifying Docker Images with Notation
+
+To ensure the authenticity and integrity of Docker images, use Notation to sign and verify the images. Here’s how you can do it:
+
+## 1. Generate a Test Key and Self-Signed Certificate:
+- To generate a test key and certificate, run the following commands:
+```bash
+notation cert generate-test --default "wabbit-networks.io"
+notation key ls
+notation cert ls
+```
+
+## 2. Sign the Docker Image:
+- Use Notation to sign the Docker image:
+```bash
+notation sign $IMAGE
+```
+- Set the image digest as an environment variable:
+```bash
+export IMAGE=myharbor-registry.online/notation-project/net-monitor@sha256:<digest>
+```
+
+## 3. Create a Trust Policy:
+- A trust policy defines which signatures are considered valid for a given registry. Save the following JSON as `trustpolicy.json`:
+```json
+{
+"version": "1.0",
+"trustPolicies": [
+{
+"name": "wabbit-networks-images",
+"registryScopes": [ "*" ],
+"signatureVerification": {
+"level": "strict"
+},
+"trustStores": [ "ca:wabbit-networks.io" ],
+"trustedIdentities": [ "*" ]
+}
+]
+}
+```
+- Import the trust policy into Notation:
+```bash
+notation policy import ./trustpolicy.json
+```
+
+## 4. Verify the Signed Image:
+- To verify the signature of an image, use the following command:
+```bash
+notation verify $IMAGE
+```
+- You can inspect the image’s signature and certificate information:
+```bash
+notation inspect $IMAGE
+```
+
```

Comment:
I would recommend add or put a link to the harbor on how to view Notary Project sigantures from Harbor portal, and how Harbor can enable the configuration, so that images not signed cannot be pulled. See https://goharbor.io/docs/main/working-with-projects/project-configuration/implementing-content-trust/#enforce-content-trust, and this is guide from harbor to use Notation https://goharbor.io/docs/main/working-with-projects/working-with-images/sign-images/#use-notation-to-sign-and-verify-artifacts-with-distribution-spec-v11-mode